src/traces/parcoords/parcoords.js

```diff
@@ -149,7 +149,10 @@ function model(layout, d, i) {
     var trace = cd0.trace;
     var lineColor = helpers.convertTypedArray(cd0.lineColor);
     var line = trace.line;
-    var deselectedLines = {color: rgba(c.deselectedLineColor)};
+    var deselectedLines = {
+        color: rgba(trace.unselected.line.color || '#777'),
```

Comment:
Can `'#777'` be the attribute default?

Reply:
We have lots of examples where a color and an opacity are specified side-by-side... and it doesn't mean it's NOT that color, just that there's opacity applied later that dilutes it.

Actually thinking about this a bit more, I might even suggest removing the "set opacity to 1 if a color is explicitly specified" and keep `'auto'` as the consistent default, because if you did want to change this color most likely you still want it diluted.

Reply:
Good call. Addressed in 47bef38.